testing/framework/TestSCons_time.py

```diff
@@ -68,6 +72,20 @@ def write_args(fp, args):
 #!/usr/bin/env python
 import os
 import sys
+
```

Comment:
What's this new blurb here do?

Reply:
it duplicates the existing stanzas for aegis and svn into one for git.  the testcase to call it doesn't exist, so it's a no-op at the moment.  git diff is breaking it in a funncy way.  I have no evidence this is needed and don't mind dropping (it's the string `git_py` and the function `write_fake_git_py)`